### ISSUE TITLE
How to debug in VScode editor

### ISSUE THREAD

**Issue by paulZzhang** [fastify/fastify-cli#71]
How to debug in VScode editor by fastify-cli

**Comment by delvedor** [comment]
Hi! I'm not sure what's the question here.
Can you rephrase?

**Comment by paulZzhang** [comment]
How to config the launch.json in VSCode editor
`{
  "version": "0.2.0",
  "configurations": [
    {
      "type": "node",
      "request": "launch",
      "name": "DEBUG",
      "program": "${workspaceFolder}\\app.js"
    }
  ]
}`
![image](https://user-images.githubusercontent.com/38898036/40151107-8a001120-59b0-11e8-8da8-1129bca011b1.png)

**Comment by delvedor** [comment]
I'm sorry, but I'm not a VSCode user, so I can't help you.
cc @jsumners @StarpTech

**Comment by paulZzhang** [comment]
@jsumners @delvedor  If I don't use fastify-cli, I can debug.

**Comment by delvedor** [comment]
@paulZzhang I don't understand your question.
`fastify-cli` does two things, bootstraps a new project with `fastify generate` or run an existing project (generated with `fastify-cli`) with `fastify start`.
You can run `fastify help <command>` to know more.

**Comment by paulZzhang** [comment]
@delvedor  I see.

**Comment by iam-pawat** [comment]
![image](https://user-images.githubusercontent.com/2102694/50597447-0a4d2c80-0eda-11e9-89e7-ebd096493cc8.png)
@paulZzhang You run your app and run debug. vs code will display process picker. you can choose your target running process what you want

**Comment by meotimdihia** [comment]
Awesome, thanks for solution @iam-pawat.

**Comment by leosuncin** [comment]
I use this configuration for `launch.json`

```jsonc
{
  // Use IntelliSense para saber los atributos posibles.
  // Mantenga el puntero para ver las descripciones de los existentes atributos.
  // Para más información, visite: https://go.microsoft.com/fwlink/?linkid=830387
  "version": "0.2.0",
  "configurations": [
    {
      "type": "pwa-node",
      "request": "attach",
      "name": "Attach by Process ID",
      "processId": "${command:PickProcess}",
      "skipFiles": ["<node_internals>/**"]
    },
    {
      "type": "pwa-node",
      "request": "launch",
      "name": "Start fastify",
      "skipFiles": ["<node_internals>/**"],
      "runtimeArgs": ["--inspect"],
      "program": "${workspaceFolder}/node_modules/fastify-cli/cli.js",
      "args": [
        "start",
        "-w",
        "--ignore-watch='test,.vscode'",
        "-l",
        "debug",
        "${workspaceFolder}/src/app.js"
      ]
    }
  ]
}
```
